extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -328,9 +332,10 @@ public void startPersistenceUnits(HibernateOrmRecorder recorder, BeanContainerBu
         recorder.startAllPersistenceUnits(beanContainer.getValue());
     }
 
-    private Optional<String> getSqlLoadScript() {
+    private Optional<String> getSqlLoadScript(LaunchMode launchMode) {
         // Explicit file or default Hibernate ORM file.
-        String script = hibernateConfig.sqlLoadScript.orElse("import.sql");
+        String script = hibernateConfig.sqlLoadScript
+                .orElseGet(() -> launchMode == LaunchMode.NORMAL ? NO_SQL_LOAD_SCRIPT_FILE : "import.sql");
```

Comment:
Hmmm, why do you need a lambda here?

Reply:
I wanted to avoid a lot of `if else` blocks and I thought lambdas were OK in deployment code, but it's gone now :)